src/watchmaker/utils/__init__.py

```diff
@@ -119,3 +121,34 @@ def clean_none(value):
         return None
 
     return value
+
+
+def set_file_perms(path, dir_mode=None, file_mode=None):
```

Comment:
maybe name this to reflect it is not operating on files?
```suggestion
def set_path_perms(path, dir_mode=None, file_mode=None):
```